test/Twig/Tests/grammarTest.php

```diff
@@ -19,6 +19,22 @@ protected function setUp()
         require_once __DIR__.'/SimpleTokenParser.php';
     }
 
+    /**
+     * Reset "xdebug.overload_var_dump" if set to 2 to avoid filename output.
+     * Reset "xdebug.cli_color" if set to On outputs ASCII color codes.
+     */
+    public static function setUpBeforeClass()
+    {
+        if (extension_loaded('xdebug')) {
+            if (ini_get('xdebug.overload_var_dump') == 2) {
```

Comment:
This has nothing to do in this PR, as it is totally unrelated.

Reply:
Hi @stof !! The problem is that I can't get the tests to pass locally, or for anyone who has `xdebug.cli_color` enabled. The `grammarTest` expects uncoloured output from `var_dump`.

I agree that this doesn't belong to this PR so your recommendation is that I make it as a separate issue?